sprintguide.rst

```diff
@@ -0,0 +1,143 @@
+.. highlight:: console
+
+.. _sprintguide:
+
+PyCon Sprint Guide
+==================
+
+This guide is written for PyCon US 2017 sprint.  For other sprints, we may need
+to come up with a different set of easy b.p.o. issues.
+
+Everyone at the sprint needs to follow the `PSF Code of Conduct <http://python.org/psf/codeofconduct/>`_.
+
+
+.. contents::
+
+Installations
+-------------
+
+The following resources take time to download.  You may not want to rely on the
+on the conference venue's wifi to download these.  Do them before you come to
+the sprint.
+
+#. Go through the :ref:`setup` guide.  Install the necessary dependencies
+   for your machine.
+
+#. Install sphinx v1.6.1::
+
+      pip install sphinx~=1.6.1
+
+#. Fork and clone the CPython repo.  Set your clone as the ``origin``, and
+   ``https://github.com/python/cpython`` as ``upstream``.
+
+#. Fork and clone the Dev Guide repo. Set your clone as the ``origin``, and
+   ``https://github.com/python/devguide`` as ``upstream``.
+
+Reading materials
+-----------------
+
+#. :ref:`Quick Start <index>` section of CPython Developer's Guide.
+
+#. :ref:`Documenting Python <documenting>`.
+
+#. :ref:`gitbootcamp`.
+
+#. :ref:`exploring`.
+
+#. :ref:`helptriage`.
+
+Others
+------
+
+#. The bug tracker is at:  https://bugs.python.org
+
+#. Create an account at the bug tracker.
+
+#. Link your GitHub account to the bug tracker account.
+
+#. Sign the :ref:`CLA <licensing>`.
+
+Communication
+-------------
+
+#. Core-mentorship `mailing list <https://mail.python.org/mailman/listinfo/core-mentorship/>`_.
+
+#. Comment on the issue, say that you're working on it. If an issue is assigned,
+   please ask first.
+
+#. #python-dev on `freenode <http://webchat.freenode.net/>`_.
+
+
+Dev Guide Issues
```

Comment:
In the past, we have had a shared document (similar to paste bin). That was the reason for my comment. And yes, usually a core dev takes an ownership to keep the doc updated. At the same other others can edit too.  (If ownership is a problem, I can volunteer to be an owner of shared doc).

I am not -1, just sharing a point that it will be a cumbersome to keep list alive.